pkgs/development/python-modules/grep-ast/default.nix

```diff
@@ -11,13 +11,13 @@
 
 buildPythonPackage rec {
   pname = "grep-ast";
-  version = "0.4.1";
+  version = "0.6.1";
```

Comment:
Just confirming that this upgrade is actually needed. Aider's author changed the API in grep-ast.